### ISSUE TITLE
images in  not imported/ rendering

### ISSUE THREAD

**Issue by slumnitz** [pysal/notebooks#77]
I noticed that most packages have a static image in their `README.md` which is not rendered in the notebook book. I suppose due to the images not being imported in this repo/ and the image links being broken since the hard-coded image path changed. 

It seems like GitHub markdown does not really support absolute image paths, so maybe it is possible to pull the images in this repo as well?

**Comment by darribas** [comment]
Good catch! Yes, I think this would have to be caught and processed on `write_package_intro`:

https://github.com/pysal/notebooks/blob/master/lib/build.py#L216

Maybe that method should also include a check for `.png` in the text and, if found, grab them?

**Comment by sjsrey** [comment]
Since we are likely to uncover other issues that should be standardized across subpackages, we should document these [here](https://github.com/pysal/submodule_template/wiki/meta-noteoboks).

**Comment by knaaptime** [comment]
+1 i was going to say something similar since we dont have any conventions currently as to where those images are stored. Would make things easy if we can just grab the `docs/images` (or whatever) dir from each package

**Comment by darribas** [comment]
Yes, it might be worth sorting out those standards first and then pulling them into the notebooks project?

**Comment by sjsrey** [comment]
There are likely to be other, similar, issues that we probably want to document in the su

> Yes, it might be worth sorting out those standards first and then pulling them into the notebooks project?

I think so. We use the notebooks project to smooth the rough edges and discover best practices. Then we codify in the submodule instructions.
